private_gpt/components/node_store/node_store_component.py

```diff
@@ -16,19 +17,38 @@ class NodeStoreComponent:
     doc_store: BaseDocumentStore
 
     @inject
-    def __init__(self) -> None:
-        try:
-            self.index_store = SimpleIndexStore.from_persist_dir(
-                persist_dir=str(local_data_path)
-            )
-        except FileNotFoundError:
-            logger.debug("Local index store not found, creating a new one")
-            self.index_store = SimpleIndexStore()
-
-        try:
-            self.doc_store = SimpleDocumentStore.from_persist_dir(
-                persist_dir=str(local_data_path)
-            )
-        except FileNotFoundError:
-            logger.debug("Local document store not found, creating a new one")
-            self.doc_store = SimpleDocumentStore()
+    def __init__(self, settings: Settings) -> None:
+        match settings.docstore.database:
+            case "local":
```

Comment:
I'd call it "simple" instead of "local" to better reflect the option being selected, given postgres can also be local

Reply:
I like simple better than local too.  You are right, this better reflects the underlying storage type.